ion-tree-list.js

```diff
@@ -18,10 +18,10 @@ function addDepthToTree(obj, depth, collapsed) {
 }
 
 function toggleCollapse(obj) {
-    for (var key in obj) {
-        if (obj[key] && typeof(obj[key]) == 'object') {
-            obj[key].collapsed = !obj[key].collapsed;
-            toggleCollapse(obj[key])
+    if (obj.tree) {
```

Comment:
Actually in my situation I have a 4 level tree with hundreds of items, when I click a branch it takes several seconds to unfold all till the leaves, which is too slow for me. Also I do not want a collapse, I want to unfold level by level which lead user to find the leaf. So I take this change so that when I click a branch, only his children will be unfolded and divs will be created, not his grandchildren. Now it moves smooth. 
So, yes, I do think it works, it cannot collapse, but user can see all his children at a glance and choose level by level down to the final item he want.